client/ayon_usd/standalone/Usd/Pinning/pinning_file_helper.py

```diff
@@ -0,0 +1,235 @@
+import json
+import os
+import re
+from typing import Dict, List
+from pxr import UsdShade, Ar, Sdf
+from urllib.parse import urlparse
+
+
+def is_uri(path):
+    parsed = urlparse(path)
+    return parsed.scheme
+
+
+# we do the assumption that the environment sets up the correct Default resolver (in AYON case this should be the AyonUsdResolver)
+resolver = Ar.GetResolver()
+
+
+def remove_root_from_dependecy_info(
+    dependecy_info: Dict[str, str], root_info: Dict[str, str]
+) -> Dict[str, str]:
+    """function that re_roots a path dict so that a Ayon_Usd_Resolver can use its own system for root replace to resolve a given path
```

Comment:
No need to mention that it's a function in docstrings.
Describe what it returns instead.

```suggestion
    """Re-roots a path dict so that a Ayon_Usd_Resolver can use its own system for root replace to resolve a given path
```

This docstring may benefit from an `example` because I have no ideea what the "path dict" is.